### DISCUSSION TITLE
[Bug report] Mercury theme (2.6.0.0): Enabled post filter shows post from the first page, only

### DISCUSSION THREAD

**Discussion by 3muchfun** [GetPublii/Publii#1361] [Bug Report] [answered]
### Operating system

macOS 13.6.4 (22G513)

### Publii version

Version: 0.44.4 (build 16557)

### Issue type

Free theme

### Bug description

My site contains of 12 posts, with having 11 posts per page. If I use the post filter, only posts from the first page are being displayed. 
If I change the number of post per page e.g. to 5, the filter only filters these 5 posts. 
I expected the filter to filter all the post on my site...

### Steps to reproduce

Have a look at fluegge.net/unterwegs and choose the filter "Dänemark", which is the 12th post.

**Comment by bobmitro** [comment]
This behavior is not a bug; it's the intended functionality. It utilizes the Isotope script to filter elements within the viewport. Only the visible items are filtered.

**Reply by 3muchfun** [reply]
Thanks for the explanation. Is there any chance to filter all of the posts by a tag? ( not via a tag page :-) )

**Reply by bobmitro** [reply]
It is, but it requires displaying all the posts on the front page.

**Reply by 3muchfun** [reply]
Bob, another question: In _Persona free_ theme there is a "show tag list" functionality, which does, what I expected from the "post filter" in _Mercury_. Anyway, is there the possibility to add the "show tag list" functionality to the Mercury theme, e.g. by editing the code?

**Reply by bobmitro** [reply]
The same functionality has been built in the Mercury theme, called "Enable post filter". So, I’m not sure what you want to move from Persona to Mercury theme.

**Reply by 3muchfun** [reply]
Ok, but using Mercury, the filter works on the Frontpage, only, as you stated above. When using Persona free, the filter brings up posts from page 3 for example, too...

**Reply by 3muchfun** [reply]
https://fluegge.net/wunderland/. Filter by "Krankenhaus" shows the post which is on page 3

**Reply by bobmitro** [reply, marked as answer]
Ahhh, I got it now. I always thought about the isotope filter that filters the posts with animation, but your example of the persona theme just displays the tag list. When you click on a tag, you are moved to the page with that tag. 
It can be done easily in the Mercury. If I have time, I'll add it there.

**Reply by 3muchfun** [reply]
Thank you, Bob!
